src/managed/OpenLiveWriter.BlogClient/Providers/BlogProvidersB5.xml

```diff
@@ -340,6 +340,36 @@
     </options>
   </provider>
   <provider>
+    <id>E83A8A63-0F40-4499-AAD1-DFB2577CCC40</id>
+    <name>Telligent Community 8.0+</name>
+    <description>The easiest and most powerful way to build and grow a robust online community.</description>
+    <clientType>Metaweblog</clientType>
+    <postApiUrl>
+      <![CDATA[
+		http://<hostname>/<groupname>/b/<blogname>/metablog
+		]]>
+    </postApiUrl>
+    <rsdEngineNamePattern>Community Server</rsdEngineNamePattern>
+    <options>
+      <supportsNewCategories>Yes</supportsNewCategories>
+      <supportsNewCategoriesInline>Yes</supportsNewCategoriesInline>
+      <useLocalTime>Yes</useLocalTime>
+      <requiresHtmlTitles>No</requiresHtmlTitles>
+      <returnsHtmlTitlesOnGet>Yes</returnsHtmlTitlesOnGet>
```

Comment:
Are you able to test some of these options against a Telligent Community 8.0+ install to see if they're still applicable to the latest version?

* Make sure publishing a post with images works (`fileUploadNameFormat`)
* Try creating at least two new categories via Open Live Writer and then publish the post. (`supportsNewCategories`, `supportsNewCategoriesInline`, `supportsMultipleCategories`)
* Try scheduling a post for a future date and time (even if its just 10min in the future) and make sure the timing works (`useLocalTime`)
* Publish a blog post where the title has angle brackets `<` in the title, then edit the post in Writer and then publish it again and make sure that the post looks correct on both the blog and in Writer (`requiresHtmlTitles` and `returnsHtmlTitlesOnGet`)
* Start a new blog post and publish as draft (`supportsPostAsDraft`)
* If you have time, you could flip <supportsEmbeds> and <supportsScripts> to Yes and try inserting a YouTube video (`supportsEmbeds`) and try inserting a GitHub gist (`supportsScripts`) and publishing that. It's possible that the policies have changed.

Thanks! If not, we can try and help recruit someone to test this.